### DISCUSSION TITLE
flv直播视频不稳定如何实现自动刷新

### DISCUSSION THREAD

**Discussion by socyli** [bytedance/xgplayer#822] [Q&A]
**您使用的西瓜播放器版本是多少？ What version of xglayer are you using?**

2.27.3
**您使用的操作系统和浏览器分别是？ What OS and browser are you using?**
windows 非IE

**如何复现问题？ What did you do?**
视频源不稳定时候

**您期望的播放器正常行为是？ What did you expect to happen?**

直播流不稳定出现丢包，如何让播放器自动刷新，而不需要人为去点刷新。
**实际播放器的表现是？ What actually happened?**

需要人为去点刷新
**可填写您所在的公司和相关产品业务，方便我们提供更好的技术支持 You can write your company and product which uses xgplayer,  for helping us provide better technical support.**
监控视频的播放

**Comment by leonardoFu** [comment]
这个不稳定反映到直播流上是什么表现，没有数据过来么？

**Comment by socyli** [comment]
有数据过来，就是可能会偶尔丢包，或者就是资源很多人访问的时候，播放器肯定拉不到流就会转圈或者出错，最后播放器就出现刷新，如何让播放器自动的一直去请求直播流。

> 这个不稳定反映到直播流上是什么表现，没有数据过来么？

**Comment by lfzs** [comment]
播放器出错、加载中都会有事件触发，通过监听这些事件，可以做一些操作，从而避免播放器中断

**Reply by superAo005** [reply]
直播流不稳定出现丢包 具体监听什么事件呢

**Comment by hongrunhui** [comment]
好像弱网情况下会触发暂停事件，但是不会触发error事件，waiting 事件偶尔会触发，这种情况如何判断自动刷新啊

**Reply by superAo005** [reply]
自动刷新是什么事件呢
